### PR TITLE
Service page content updates

### PR DESCRIPTION
**Pull request by SimonSaysAutomate**
Content changes to COO, Automation, HubSpot and Pipedrive & also added quotation box styling.

### CODE REVIEW

File: _services/automation.html [hunk 1]
```diff
@@ -7,43 +7,32 @@
 meta: "Unlock your business's potential with tailored automation. We streamline operations using HubSpot, Pipedrive, Zapier, and AI, driving revenue growth while reducing manual tasks efficiently."
 excerpt: "Unlock tailored business process automation strategies to help your business drive sustainable growth through the power of automation. We work with you to identify bottlenecks and resolve them using powerful automation tools like HubSpot, Pipedrive and Zapier."
 header-img: "/img/blank-notepad.jpg"
-top-section: "<p>Automation isn’t a one-size-fits-all solution, a tailored strategy will help your business unlock its full potential.</p>
-<p>That’s where our Business Process Automation expertise makes all the difference.</p>"
+top-section: "<p>Ready to transform your business through automation?
+</p>
+<p>Automating processes isn’t a one-size-fits-all solution, it requires a tailored strategy and expertise. We work together with you to analyse your processes and create an actionable strategy deployed via using services like Zapier, HubSpot and Pipedrive.
```
Comment (maceq687) [inline _services/automation.html]: Let's try to unify the indentation in this and above paragraph

File: _services/automation.html [hunk 2]
```diff
@@ -7,43 +7,32 @@
...
+<p>
+  Proper implementation will save your team time, grow your revenue and grow your business.
+</p>"
 ---
 
 <div class="content-row">
   <section class="content-row__section">
     <img src="/img/build-icon.svg" class="content-row__icon" alt="" />
-    <h2 class="content-row__title">Our Services</h2>
+    <h2 class="content-row__title">How we help your business become more efficent</h2>
```
Comment (maceq687) [inline _services/automation.html]: 'efficient'

File: _services/automation.html [hunk 3]
```diff
@@ -7,43 +7,32 @@
...
-        Automation Tools like Pipedrive’s Sales Automation System, Workflow
-        Automation Platforms like Zapier, Email and Communication Automation tools
-        like Pipedrive’s Campaigns feature and HubSpot’s Workflow Automations, and
-        more.
-      </p>
-      <p>
-        By leveraging these advanced technologies we streamline your business
-        processes, reduce manual tasks, and drive sustainable growth—saving you
-        valuable time and resources.
+        Getting your tech stack to work together can be problematic, but using AI and automation tools like <a href="https://deranged.dk/pipedrive">Pipedrive</a>, <a href="https://deranged.dk/hubspot">HubSpot</a>, and Zapier we can efficently integrate your software and streamline your workflows saving you valuable time and resources.
```
Comment (maceq687) [inline _services/automation.html]: 'efficiently'

File: _services/automation.html [hunk 4]
```diff
@@ -57,43 +46,34 @@ <h2 class="content-row__title">Our Services</h2>
...
+        <strong>On-demand support:</strong> We offer the flexibility of guidance whenever you need it - without the long-term commitment or costs of a full-time hire.
+      </p>
+      <p>
+      <strong>Expert advice:</strong> With extensive automation backgrounds and experience we deliver functional automations.
       </p>
       <p>
-        We have run, and automated, our own successful companies - we know
-        firsthand the common pitfalls businesses encounter. Our expertise allows
-        us to help you avoid those obstacles through smart automation strategies.
+      <strong>Certified technical expertise:</strong> We partner with RevOps Software vendors
```
Comment (maceq687) [inline _services/automation.html]: Let's try to unify indentation and line brakes in this section

File: _services/automation.html [hunk 5]
```diff
@@ -57,43 +46,34 @@ <h2 class="content-row__title">Our Services</h2>
...
-        automation framework that ensures long-term success and sustainable
-        growth.
+       <strong>Experience:</strong> We have run, and automated, our own successful companies - we know firsthand the common pitfalls businesses encounter. We help you avoid those obstacles through smart automation strategies.
       </p>
       <p>
-        Plug and Play Expertise. We hit the ground running, quickly integrating
-        into new companies and optimizing business processes through automation.
-        Our ability to automate and streamline processes helps you achieve results
-        fast.
+        <strong>Scaleable:</strong> We specialise in working with startups and scaleups and understand the importance of early growth stages. We’ll help you build a scalable
```
Comment (maceq687) [inline _services/automation.html]: 'scalable'

Comment (maceq687) [inline _services/automation.html]: 'scale-ups' 🤔

File: _services/coo.html [hunk 6]
```diff
@@ -7,30 +7,44 @@
...
-        professionals who seamlessly integrate with your existing team, bringing a
-        wealth of knowledge and strategic insight to your organization from day
-        one. We focus on the areas that matter most to your business, from
-        business process improvements to day to day operational management to help
-        you keep your business running efficiently.
+        ➟ Struggling to focus on the areas that matter the most to your business? Or do you need support in driving revenue? - we can help.
       </p>
-      <p>Our key services include:</p>
+        <p>
+        Our experienced professionals seamlessly integrate with your existing team from day one bringing a wealth of knowledge and strategic insight to your organisation alongside:
```
Comment (maceq687) [inline _services/coo.html:30]: Let's try unifying the indentation here

File: _services/coo.html [hunk 7]
```diff
@@ -47,40 +61,32 @@ <h2 class="content-row__title">Our Services</h2>
...
-        operational workflows, manage key projects, and drive sustainable growth.
+        <strong>Access to our expertise:</strong> By partnering with us, you benefit from our years of expertise in business that translates into our ability to streamline your operational workflows, manage key projects, and drive sustainable growth from day one.
       </p>
       <p>
-        Extensive Industry Experience: Having owned, operated and worked at
-        startups, scaleups and large organizations we bring a deep understanding
-        of the internal and external challenges businesses face.
-      </p>
+        <strong>Cost-effective and flexible:</strong> Access operational leadership on-demand, leadership that might otherwise be inaccessible to startups and scaleups without the long-term commitment or costs associated with a full-time COO hire.
+    </p>
```
Comment (maceq687) [inline _services/coo.html]: Let's try unifying the indentation here

File: _services/coo.html [hunk 8]
```diff
@@ -47,40 +61,32 @@ <h2 class="content-row__title">Our Services</h2>
...
+      <strong>Certified technical expertise:</strong> We partner with RevOps Software vendors
+        like <a href="/hubspot">HubSpot</a> and <a href="/pipedrive">Pipedrive</a> to
+        implement their solutions for our clients and maintain them.
       </p>
       <p>
-        Strategic and Operational Support: We can help transform your
-        organization's vision into actionable plans, to help support sustainable
-        and scalable growth.
+        <strong>Industry Experience:</strong> Having owned, operated and worked at
+        startups, scaleups and large organizations we bring a deep understanding
```
Comment (maceq687) [inline _services/coo.html]: 'scale-ups' 🤔

File: _services/hubspot.html [hunk 9]
```diff
@@ -10,10 +10,13 @@
...
-top-section: "<p>We are Official HubSpot Solution Providers</a>
-- and offer comprehensive HubSpot services. We use HubSpot ourselves and can
-help you integrate your tech stack with HubSpot so you can save your
-business time, effort and money.</p>"
+top-section: "<p>Are you having trouble getting the best out of HubSpot? Or are you looking to improve your sales processes and drive revenue growth?</p>
+<p>
+With 5+ years experience with HubSpot, we've helped businesses like yours increase efficiency, drive growth and saved them time, effort and money.
+</p>
+<p>
+  We are official <strong>HubSpot Solution Providers</a></strong> - and work across the entire CRM platform.
```
Comment (maceq687) [inline _services/hubspot.html:18]: Let's unify the indentations here

File: _services/hubspot.html [hunk 10]
```diff
@@ -38,29 +41,38 @@
 <div class="content-row">
   <section class="content-row__section">
     <img src="/img/build-icon.svg" class="content-row__icon" alt="" />
-    <h2 class="content-row__title">Our Services</h2>
+    <h2 class="content-row__title">Solution Providers</h2>
     <div class="content-row__text">
-      <p>We offer the following services in both English & Danish:</p>
+      <p>
+      Our unique approach focuses on identifying solutions for your pain points and bringing value from day one. We work in an agile manner to ensure that your CRM is functional quickly - and working for your benefit.
```
Comment (maceq687) [inline _services/hubspot.html:47]: Let's unify the indentations in this section

File: _services/hubspot.html [hunk 11]
```diff
@@ -38,29 +41,38 @@
...
       <p>
         <strong>
-          Tailor your HubSpot CRM to fit your organization's needs with our
-          professional support
+          Ready to transform your HubSpot experience?</strong> - We'll tailor HubSpot to fit your organisation's needs with our professional support.
         </strong>
       </p>
-    </div>
+      <center>
+        <div class="flex-row">
```
Comment (maceq687) [inline _services/hubspot.html]: There is no `flex-row` class in services

File: _services/hubspot.html [hunk 12]
```diff
@@ -76,78 +88,91 @@ <h2 class="content-row__title">Our Services</h2>
...
-        HubSpot is a platform that encompasses everything scaling companies need,
-        from Sales and Marketing to Customer Service and Content Creation,
-        delivering a crafted, not cobbled approach to your tech stack.
+        HubSpot's platform encompasses everything scaling companies need, delivering a crafted, not cobbled approach to your tech stack that covers everything from sales and marketing to customer service and content creation.
       </p>
-      <p>
-        HubSpots tools, which are powerful alone, but better together, offer a
-        cohesive and customizable solution that helps your team grow and offer a
-        lower cost of ownership combined than cobbled together solutions.
+      <p>It is highly customisable and helps your team grow while offering a lower cost of ownership than diverse tech stacks.
```
Comment (maceq687) [inline _services/hubspot.html]: Let's try to unify the indentation in this section

File: _services/hubspot.html [hunk 13]
```diff
@@ -76,78 +88,91 @@ <h2 class="content-row__title">Our Services</h2>
...
-        >
+        We will:
       </p>
       <p>
-        <strong>Marketing Hub</strong> - Marketing software powered by AI helping
-        you generate leads and automate your marketing campaigns.
-        <a href="https://www.hubspot.com/products/marketing" target="_blank"
-          >Read More</a
-        >
+        <ul>
```
Comment (maceq687) [inline _services/hubspot.html:123]: there is no need to nest unordered list `<ul></ul>` inside paragraph `<p></p>` tags

File: _services/hubspot.html [hunk 14]
```diff
@@ -76,78 +88,91 @@ <h2 class="content-row__title">Our Services</h2>
...
       <p>
-        <strong>Service Hub</strong> - Harness the power of AI to help scale your
-        support operations.
-        <a href="https://www.hubspot.com/products/service" target="_blank"
-          >Read More</a
-        >
+        <strong>Breeze AI</strong> - AI made easy -
+        an AI solution that delivers ease, speed, and unification across your business.
+        <a href="https://www.hubspot.com/products/artificial-intelligence" target="_blank"
+          >Read More</a>
```
Comment (maceq687) [inline _services/hubspot.html]: We can discuss if it make sense from SEO perspective to replace 'Read more' links text here

File: _services/pipedrive.html [hunk 15]
```diff
@@ -10,8 +10,12 @@
 meta: "Premier Pipedrive Solution Providers. We offer onboarding, CRM setup, automation, and support in English & Danish, to optimize Pipedrive for your business."
 excerpt: "We are here to support your Pipedrive journey with expert implementation, onboarding and sales process optimisation. Our partnership with Pipedrive allows us to guide your team through every step - from decision making to set up. We drive your success with Pipedrive."
 header-img: "img/entrepreneur-on-sofa.jpg"
-top-section: "<p>We are Official Premier Pipedrive Solution Providers - and want to help you succeed with Pipedrive - The easy and effective CRM for closing deals.</p>
-<p>We have vast experience with Pipedrive’s Sales and Marketing focused CRM and are ready to help you set you on the right track to get the maximum value out of Pipedrive.</p>"
+top-section: "
+<p>Are you looking for a dedicated sales CRM? Or are you having trouble getting the best out of Pipedrive?
+</p>
+  With 5+ years Pipedrive experience we've helped businesses like yours streamline their sales processes & automate repetitive tasks - saving them time and driving growth.
```
Comment (maceq687) [inline _services/pipedrive.html:16]: Missing paragraph opening tag here, and let's try to unify the indentation here

File: _services/pipedrive.html [hunk 16]
```diff
@@ -51,30 +55,40 @@
...
-        <li>Pipedrive Campaigns Setup</li>
-      </ul>
       <p>
-        We can also help you to migrate from your pre-existing CRM system to
-        Pipedrive alongside setting up Pipedrive automations and reporting so you
-        can visually manage your contact and lead information all in one powerful
-        platform alongside automating mundane tasks.
+        Our unique approach to Pipedrive focuses on identifying solutions for your company pain points and bringing value from day one. We work in an agile manner to ensure that your CRM is functional quickly - and working for your benefit.
+      </p>
+        <p>We can help you implement:</p>
```
Comment (maceq687) [inline _services/pipedrive.html]: Let's try to unify indentation in this section

File: _services/pipedrive.html [hunk 17]
```diff
@@ -51,30 +55,40 @@
...
-          Whether you're new to Pipedrive or not, let us expertly set up your CRM
-          to suit your organization's needs.
-        </strong>
+        We specialise in working with international companies - In both English & Danish - and aim to establish long term relationships with our clients.
+      </p>
+      <p> 
+        <strong>Ready to transform your Pipedrive experience?</strong> - We'll tailor Pipedrive to fit your organisation's needs with our professional support.
+      </p>
+        <center>
+          <div class="flex-row">
```
Comment (maceq687) [inline _services/pipedrive.html]: There is no `flex-row` class in services

File: _services/pipedrive.html [hunk 18]
```diff
@@ -88,22 +102,27 @@ <h2 class="content-row__title">Our Services</h2>
...
+      </p>
+      <p>
+        Combined with intelligent automations and a built in AI Sales Assistant, Pipedrive is your go to tool for managing sales leads and deals.
       </p>
+      <section class="customer-quote-section customer-quote-section--big py-4">
+        <p class="quote-text">
+          Used by over 100,000 companies in 179 countries
+        </p>
+        <p class="quotee"><a href="https://www.pipedrive.com/en/crm-comparison/best-crm">Source: Pipedrive</a></p>
+    </section>
```
Comment (maceq687) [inline _services/pipedrive.html]: Let's unify indentation here

File: _services/pipedrive.html [hunk 19]
```diff
@@ -120,35 +139,58 @@ <h2 class="content-row__title">Why Choose Pipedrive</h2>
...
+    <h2 class="content-row__title">Setup</h2>
     <div class="content-row__text">
       <p>
-        Pipedrive is a feature rich CRM, here are some of the key components that
-        will help you optimize your business.
+        Pipedrive is packed full of useful features, but you've got to start somewhere - we can help you decide what to implement first.
       </p>
+      <p>
+        We will:
+        <p>
```
Comment (maceq687) [inline _services/pipedrive.html]: Something went wrong with paragraph tags order here

Comment (maceq687) [inline _services/pipedrive.html]: Also let's try fixing indentation here

File: _services/pipedrive.html [hunk 20]
```diff
@@ -120,35 +139,58 @@ <h2 class="content-row__title">Why Choose Pipedrive</h2>
...
-        <strong>Unlimited sales pipelines on all plans</strong> - If your
-        organization has multiple sales processes, Pipedrive lets you manage them
-        with ease.
+        <strong>Custom fields</strong> - Add the data that reflects
+        your sales cycle to your leads and deals.
       </p>
       <p>
-        <strong>Custom fields on all plans</strong> - Add the data that reflects
-        your sales cycle to your leads and deals.
+      <Strong>Email templates</Strong> - Helps you maximise efficiency by streamlining emails.
```
Comment (maceq687) [inline _services/pipedrive.html]: We always keep html tags lowercase

Comment (maceq687) [inline _services/pipedrive.html]: Let's fix indentation here

**Review by maceq687 — CHANGES_REQUESTED** [review]
Couple of observation regarding indentation, mistakes using html tags and possible spelling issues:

File: _services/hubspot.html (hunk 14 above)
Comment (SimonSaysAutomate) [reply _services/hubspot.html]: Changed to link in titles.

**Review by maceq687 — APPROVED** [review]
Looks good now

**Pull request merged**